README.md

```diff
@@ -10,7 +10,22 @@ for OpenShift Console.
 
 ## Local development
 
-### Option 1 (recommended): Docker + VSCode Remote Container
+### Option 1 (recommended):
+
+Open two terminals and navigate to the kubevirt-plugin directory in both of them. The first terminal will run a containerized instance of console and the second will run the kubevirt-plugin.
+
+In the first terminal:
+
+1. Log into the OpenShift cluster you are using.
+2. Run `yarn start-console` OR `./start-console.sh` OR `./start-console-auth-mode.sh`.
+
+In the second terminal:
+
+1. Run `yarn && yarn dev`
+
+NOTE: `./start-console-auth-mode.sh` is when login page is needed.
```

Comment:
start-console start ignore authentication and auth-mode isn't ignoring 

maybe?
NOTE: `./start-console-auth-mode.sh` is when authentication is needed, start-console.sh, ignores authentication.